src/client/content_cao.cpp

```diff
@@ -557,6 +556,10 @@ void GenericCAO::removeFromScene(bool permanent)
 		clearParentAttachment();
 	}
 
+	ShadowRenderer *shadow = RenderingEngine::get_shadow_renderer();
```

Comment:
this assigment must be done in condition as shadow is not used after

Reply:
fixed